tests/integration/test_charm_metrics_success.py

```diff
@@ -59,6 +59,10 @@ async def test_charm_issues_runner_installed_metric(
     """
     await instance_helper.ensure_charm_has_runner(app)
 
+    # Set the number of virtual machines to 0 to speedup reconciliation
+    await app.set_config({VIRTUAL_MACHINES_CONFIG_NAME: "0"})
+    await reconcile(app=app, model=model)
```

Comment:
We need to reconcile because the runner installed metric is now issued in the next reconciliation and no longer directly in the build function (because the installed timestamp now lives in cloud-init).